### PR TITLE
Upgrading Jackson-Databind version

### PR DESCRIPTION
**Pull request by Rishikesh1159**
Signed-off-by: Rishikesh1159 <rishireddy1159@gmail.com>

### Description
Update the version of jackson-databind to remove the vulnerability [WS-2021-0616](https://www.whitesourcesoftware.com/vulnerability-database/WS-2021-0616)
 
### Issues Resolved
None
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: buildSrc/build.gradle [hunk 1]
```diff
@@ -116,7 +116,7 @@ dependencies {
   api 'com.avast.gradle:gradle-docker-compose-plugin:0.14.12'
   api 'org.apache.maven:maven-model:3.6.2'
   api 'com.networknt:json-schema-validator:1.0.36'
-  api 'com.fasterxml.jackson.core:jackson-databind:2.12.5'
+  api 'com.fasterxml.jackson.core:jackson-databind:2.12.6'
```
Comment (reta) [inline buildSrc/build.gradle:119]: Could you please change it to`${props.getProperty('jackson')}` ? Thank you.

**Comment by reta** [comment]
@Rishikesh1159 please run `gradle updateSHAs`, there are SHAs updates for artifacts, thank you.

File: buildSrc/build.gradle (hunk 1 above)
Comment (Rishikesh1159) [reply buildSrc/build.gradle:119]: @reta Sure I will change that

**Review by reta — APPROVED** [review]

**Comment by Rishikesh1159** [comment]
start gradle check

**Review by saratvemulapalli — APPROVED** [review]

**Pull request merged**